app/assets/components/button/_button.scss

```diff
@@ -0,0 +1,4 @@
+.nhsuk-button {
+  height: 58px;
+  padding: 0 37px 0 37px;
```

Comment:
Can we use the built-in spacing classes for this please?
https://service-manual.nhs.uk/design-system/styles/spacing

The benefit being that they are responsive across mobile and desktop, and should adapt to the user's preferences (screen size, text size, etc). Fixed padding does not offer the same behaviour.

For this, I would try something like: `padding: 0 nhsuk-spacing(6);`